conformance/02_push_test.go

```diff
@@ -9,6 +9,10 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+const (
+	emptyLayerTestTag = "emptylayer"
+)
```

Comment:
Can we put this with other constants in setup.gp?

Reply:
Sure